src/interface/interface.vue

```diff
@@ -307,6 +307,7 @@
     }
 
     /* Editor Styles */
+    .flexible-editor :deep(.ProseMirror li > * ~ *),
```

Comment:
I think we should make this more specific and add more complex selectors, to target some edge cases:

```css
.flexible-editor .ProseMirror li > * ~ :not(ul):not(ol),
.flexible-editor .ProseMirror li > * ~ :not(ul):not(ol) ~ *,
```

The above code will only add the margin to sibling children that are not ul/ol elements, unless one already has a margin.

Before:
![before](https://github.com/user-attachments/assets/52155394-b0c1-41d8-a6f0-bc9a4d90eebf)

After:
![after](https://github.com/user-attachments/assets/142fc963-2a17-492a-ab3c-d1eaa4f434b0)

I think it looks cleaner.

Reply:
@gbicou Want to update your PR?

Reply:
updated